src/main/java/Itstime/planear/schedule/Controller/ScheduleController.java

```diff
@@ -0,0 +1,35 @@
+package Itstime.planear.schedule.Controller;
+
+import Itstime.planear.common.ApiResponse;
+import Itstime.planear.schedule.DTO.ScheduleRequestDTO;
+import Itstime.planear.schedule.DTO.ScheduleResponseDTO;
+import Itstime.planear.schedule.Service.ScheduleService;
+import jakarta.servlet.http.HttpSession;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@Slf4j
+public class ScheduleController {
+    private final ScheduleService scheduleService;
+
+    @PostMapping("/schedule")
+    public ResponseEntity<?> create(@RequestBody ScheduleRequestDTO.ScheduleCreateDTO scheduleCreateDTO, HttpSession httpSession) {
```

Comment:
```suggestion
    public ResponseEntity<ScheduleResponseDTO.ScheduleCreateDTO> create(@RequestBody ScheduleRequestDTO.ScheduleCreateDTO scheduleCreateDTO, HttpSession httpSession) {
```
요렇게 하면 generic 쓸 수 있어!

Reply:
옙 ! 수정하겠습니다